### PR TITLE
Fix/test unit

### PR DESCRIPTION
**Pull request by Volubyl**
## Detailed purpose of the PR

1 Increased code coverage but still under the threshold  for those files :

- layer/data/core.js (will work on it)
- reducers/navigations.js
-containers/card-scalable.js
- containers/video-controlable.js 
-  containers/with-layout.js  

2. Fix the unit tests

### Result and observation

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra dependency ?**
      If checked, Which extra dependency did you add ? (name, purpose, link ...).

### Testing Strategy

- [ ] Already covered by tests
- [ ] Manual testing
- [X] Unit testing

### CODE REVIEW

File: jest.config.unit.js [hunk 1]
```diff
@@ -15,7 +15,13 @@ module.exports = {
   setupFiles: ['./jest-setup.js'],
   collectCoverage: true,
   coverageReporters: ['lcov', 'text', 'html'],
-  coveragePathIgnorePatterns: ['node_modules', 'jest-setup.js', 'src/*.js'],
+  coveragePathIgnorePatterns: [
+    'node_modules',
+    'jest-setup.js',
+    'src/*.js',
+    '<rootDir>/src/layer/data/progressions.js',
```
Comment (Volubyl) [inline jest.config.unit.js]: only temporary until those two function are not fully implementated using the async storage

File: jest.config.unit.js [hunk 2]
```diff
@@ -29,12 +35,6 @@ module.exports = {
       lines: 100,
       statements: 100
     },
-    'src/redux': {
```
Comment (Volubyl) [inline jest.config.unit.js]: Was a duplicate entry

Comment (VincentCATILLON) [reply jest.config.unit.js]: There is no `src/redux` config

File: src/__fixtures__/media.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const media: Media = {
       _id: '1',
       mimeType: 'image/png',
       url:
-        'https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwijhPuyoZPgAhWy34UKHR_RAX0QjRx6BAgBEAU&url=http%3A%2F%2Ffreesoftwaremagazine.com%2Farticles%2Fblender_foundation_big_buck_bunny_peach%2F&psig=AOvVaw2B33-_n-yj3Lc5jnb2xP0r&ust=1548860557455165'
+        'https://www.coorpacademy.com/wp-content/uploads/2019/01/Difficult-Conversations-cours.jpg'
```
Comment (VincentCATILLON) [inline src/__fixtures__/media.js]: 👍

File: src/components/question.js [hunk 4]
```diff
@@ -80,6 +80,7 @@ const Question = ({
         <View>
           {media.src && (
             <ImageBackground
+              style={{width: '100%', height: 50}}
```
Comment (VincentCATILLON) [inline src/components/question.js]: Should be `style={styles.image}` and `styles.image` should be :
```
{
  ...
  image: {
    height: 200
  }
}
```

File: jest.config.unit.js (hunk 2 above)
Comment (VincentCATILLON) [reply jest.config.unit.js]: Oh ok, it's a bad copy paste from me. please rename to `src/layer` please ^^

**Comment by VincentCATILLON** [comment]
Good job dude

**Pull request merged**